packaging/ansible-runner-service-project/project/roles/ovirt-host-deploy-ovn-certificates/tasks/main.yml

```diff
@@ -69,6 +69,15 @@
       mode: 0440
       remote_src: yes
 
+  - name: Populate service facts
+    service_facts:
+
+  - name: Restart OVN service
```

Comment:
lgtm, @almusil @erav please approve

Reply:
I have not realized it earlier, but we need to also restart `openvswitch` and `openvswitch-ipsec`. As those certifacates are used there too.

Reply:
Ack, will add them.